app/views/catalog/_zero_results.html.erb

```diff
@@ -1,5 +1,10 @@
 <h1><%= t('trln_argon.search.zero_results.title', local_institution_name: institution_long_name) %></h1>
 
+<% if filters_applied? %>
+   <% remove_applied_facets %>
```

Comment:
This is buggy. You are removing the f parameter from the search state even if the user does not click the link to remove the search state. The result is that if you end up on the no results page and run another search from that page the facets are removed even if that was not your intent.

Reply:
See comment on next line for a suggested fix,